.github/workflows/ci-cd.yml

```diff
@@ -38,11 +38,10 @@ jobs:
         python -m pytest --continue-on-collection-errors --junit-xml ../test-results/pytest.xml
 
     - name: Unit Test Results
-      uses: EnricoMi/publish-unit-test-result-action@master
+      uses: ./
```

Comment:
Getting this to work across branches and forks is great, this might even fix #45.

Reply:
yup, likely will 😃